docs/manage/data/permissions.md

```diff
@@ -0,0 +1,31 @@
+This includes data from any resources of locations shared with organizations you own.
+
+Users have different capabilities when it comes to managing data, depending on whether they are [an owner or an operator](/manage/fleet/#permissions) of a given {{< glossary_tooltip term_id="organization" text="organization" >}}, {{< glossary_tooltip term_id="location" text="location" >}}, or {{< glossary_tooltip term_id="robot" text="robot" >}}.
+The following table summarizes the permissions for owners and operators at different levels:
+
+| Permissions                         | Org owner | Org operator | Location owner                                | Location operator | Robot owner                                   | Robot operator |
+| ----------------------------------- | --------- | ------------ | --------------------------------------------- | ----------------- | --------------------------------------------- | -------------- |
+| View data                           | **Yes**   | No           | **Yes\***                                     | No                | **Yes\*\***                                   | No             |
+| See data tags                       | **Yes**   | No           | Only tags applied to data they have access to | No                | Only tags applied to data they have access to | No             |
+| Edit data (add tags, delete info)   | **Yes**   | No           | **Yes\***                                     | No                | **Yes\*\***                                   | No             |
+| Train models                        | **Yes**   | No           | **Yes** on data they have access to           | No                | **Yes** on data they have access to           | No             |
+| Upload models/packages              | **Yes**   | No           | **Yes\***                                     | No                | **Yes**                                       | No             |
+| View organization models/packages   | **Yes**   | No           | **Yes**                                       | No                | **Yes**                                       | No             |
+| Use organization models/packages    | **Yes**   | No           | **Yes**                                       | No                | **Yes**                                       | No             |
+| Delete models/packages              | **Yes**   | No           | No                                            | No                | No                                            | No             |
+| Export data with the CLI or the app | **Yes**   | No           | **Yes\***                                     | No                | **Yes\*\***                                   | No             |
+
+\* For data from the location they have access to
+
+\*\* For data from the robot
```

Comment:
I think this part should probably replace the content here: https://docs.viam.com/manage/fleet/#permissions